internal/ncproxyttrpc/networkconfigproxy.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+option go_package = "github.com/Microsoft/hcsshim/internal/ncproxyttrpc/;ncproxyttrpc";
+
+import weak "gogoproto/gogo.proto";
+
+service NetworkConfigProxy {
```

Comment:
It would probably be preferable to combine the TTRPC and GRPC `NetworkConfigProxy` service definitions, and generate both TTRPC and GRPC code from that single proto file. Is that something we can do?

Reply:
I'm not sure yet. Right now if you specify the ttrpc plugin in protobuild.toml it will only generate code usable by ttrpc specifically. The grpc package isn't even imported and is replaced with `github_com_containerd_ttrpc "github.com/containerd/ttrpc"`. There MAY be a way to get the best of both worlds and have ttrpc/grpc services and clients generated to one pb.go file off of the same proto definitions but I'll have to screw around with this to see.

Reply:
I've searched around but I think it's either just one or the other if you specify the plugin, can't find a way to generate both ttrpc and grpc interfaces in one file :/

Reply:
Ok this is actually possible, you just need to specify "grpc, ttrpc" in the overrides in protobuild.toml